src/Directory.Build.targets

```diff
@@ -26,4 +26,99 @@
       <Private>false</Private>
     </ProjectReference>
   </ItemDefinitionGroup>
+
+  <!-- Using Inputs/Outputs just to loop over test architectures -->
+  <Target Name="InnerTest" Inputs="*%(_TestArchitectureItems.Identity)" Outputs="*%(_TestArchitectureItems.Identity)" Returns="@(_FailedTestRuns)">
+
+    <PropertyGroup>
+      <_TestArchitecture>%(_TestArchitectureItems.Identity)</_TestArchitecture>
+      <_TestEnvironment>$(TargetFramework)|$(_TestArchitecture)</_TestEnvironment>
+      <_TestOutPathNoExt>$(ArtifactsTestResultsDir)$(MSBuildProjectName)_$(TargetFramework)_$(_TestArchitecture)</_TestOutPathNoExt>
+      <_TestStdOutPath>$(_TestOutPathNoExt).log</_TestStdOutPath>
+      <_TestResultsXmlPath>$(_TestOutPathNoExt).xml</_TestResultsXmlPath>
+      <_TestResultsHtmlPath>$(_TestOutPathNoExt).html</_TestResultsHtmlPath>
+      <_TestResultsOpenCoverPath>$(_TestOutPathNoExt).coverage</_TestResultsOpenCoverPath>
+      <_RunOnCore>false</_RunOnCore>
+      <_RunOnCore Condition="$(TargetFramework.StartsWith('netcoreapp'))">true</_RunOnCore>
+      <_UseOpenCover>$(UseOpenCover)</_UseOpenCover>
```

Comment:
Who/where is this set?

Reply:
It's passed in using `/p:UseOpenCover` in **build.ps1**.

Reply:
Nevermind, figured it out.